marklogic/documents.py

```diff
@@ -272,9 +273,17 @@ def _extract_values_from_header(part) -> dict:
 
     content_type = None
     if part.headers.get("Content-Type".encode(encoding)):
-        content_type = part.headers["Content-Type".encode(encoding)].decode(encoding)
+        content_type = part.headers["Content-Type".encode(encoding)].decode(
+            encoding
+        )
+
+    content_disposition_header = part.headers[
+        "Content-Disposition".encode(encoding)
+    ].decode(encoding)
+    msg = Message()
+    msg["content-disposition"] = content_disposition_header
+    uri = msg.get_filename()
 
-    uri = disposition_values["filename"]
     if uri.startswith('"'):
```

Comment:
Test to see if this block is still needed - I am guessing that the `Message` class will remove the double quotes automatically.